Makefile

```diff
@@ -274,7 +274,7 @@ web-build: ng ## build the web components.
 .PHONY: build
 build: bin/app ## build the yunikorn-history-server binary for current OS and architecture.
 	echo "Building yunikorn-history-server binary for $(OS)/$(ARCH)"
-	GOOS=$(OS) GOARCH=$(ARCH) $(GO) build -o $(LOCALBIN_APP)/yunikorn-history-server 										\
+	CGO_ENABLED=0 GOOS=$(OS) GOARCH=$(ARCH) $(GO) build -o $(LOCALBIN_APP)/yunikorn-history-server 										\
```

Comment:
Hi, What is the particular reason for disabling CGO? I don't think it is related to the failure. Could you provide a little bit more details please?

Reply:
Hi @sudiptob2, thanks for review.
Binary file in the alpine is not executable without disabling CGO. After setting this option, it is executable.
Current version.
![image](https://github.com/user-attachments/assets/c4bb5759-dedf-41da-997e-7c626bad4fb9)
![image](https://github.com/user-attachments/assets/8a64660a-5fac-4af6-ac99-f7112d912a83)
Modified version.
![image](https://github.com/user-attachments/assets/bf47c8bb-6547-47f2-832c-3741de4cb9fe)
[Ref](https://stackoverflow.com/questions/36279253/go-compiled-binary-wont-run-in-an-alpine-docker-container-on-ubuntu-host)

Reply:
Hi @sudiptob2. 
My machine setting is following.
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Arch:  x86_64

Reply:
@0yukali0 @sudiptob2 I tested on my Ubuntu 22.04.4 LTS system (x86_64 arch) and was seeing the error - it does not build the `bin/app/yunikorn-history-server` - the solution of using `CGO_ENABLED=0` does work. I found out from reading

  https://stackoverflow.com/questions/61515186/when-using-cgo-enabled-is-must-and-what-happens

that CGO is disabled by default when cross-compiling (which is why we didn't see it when doing our builds on macOS/arm64) but enabled by default on native compilations. I think this PR is good.